src/manifest.json

```diff
@@ -13,6 +13,7 @@
     "<all_urls>",
     "activeTab",
     "cookies",
+    "browsingData",
```

Comment:
suggestion (blocking): This will trigger a new permission prompt "Clear recent browsing history, cookies, and related data" during the update. To avoid this scary permission prompt in previous updates, we've moved this to `optional_permissions` and then only asked for the permission when the user tried to use the new feature.

Can you try a similar approach here to see how hard or easy it might be to do this?

8.1.3 permissions:
<img width="443" alt="image" src="https://github.com/user-attachments/assets/101811fb-3085-44bd-afd6-b6a5f74ce364">

This branch permissions:
<img width="445" alt="image" src="https://github.com/user-attachments/assets/47ae6cba-6406-45ab-80ef-5ea4c2a71755">

Reply:
Added this into https://github.com/mozilla/multi-account-containers/pull/2654/commits/c644a60e4609defdd96191c205cd25cbef4fcca0